packages/block-editor/src/store/resolvers.js

```diff
@@ -0,0 +1,6 @@
+export const getFetchedPatterns =
+	( fetch ) =>
```

Comment:
If I'm not wrong, "fetch" comes from the store itself, so why not select it instead?

Reply:
On a related note, what happens when the `__exFetchBlockPatterns` setting changes? Do we detect that and invalidate/refetch the resolved selector state, or ignore the change and resolve only once, with the "fetch" function that was present at the time?

Reply:
It was a remnant from old code, will adjust it

Reply:
Adjusted :) I also added `shouldInvalidate`.